APPS-730 Clean Up Payment Method API (#551)
spryker/spryker-docs: docs/scos/user/technology-partners/202108.0/payment-partners/bs-payone/scos-integration/payone-integration-into-the-scos-project.md

}
```

> 13. Add checkout plugins of Payone in `Pyz\Zed\Checkout\CheckoutDependencyProvider`:

Reviewer: ```suggestion
13. Add the Payone checkout plugins to `Pyz\Zed\Checkout\CheckoutDependencyProvider`:
```
